StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1137ElementsShouldHaveTheSameIndentation.cs

```diff
@@ -480,6 +480,12 @@ private static void CheckBraces(SyntaxNodeAnalysisContext context, SyntaxToken o
                 return;
             }
 
+            if (!openBraceToken.IsFirstInLine())
```

Comment:
❓ Should we still validate this, but validate it against the first token on the line containing the `{`?